extensions/wikia/Helios/User.class.php

```diff
@@ -196,6 +196,22 @@ public static function clearAccessTokenCookie() {
 			time() - self::ACCESS_TOKEN_COOKIE_TTL,
```

Comment:
@garthwebb do you remember why we use ```time() - self::ACCESS_TOKEN_COOKIE_TTL``` instead of simply ```-1``` to expire immediately?